### PR TITLE
touch up width for datepicker

### PR DESCRIPTION
**Pull request by M00staff**
picked up by testing - match the date picker width to the other filtered fields

image

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
Should the width maybe be a prop, cause if we use this on another page as a filter, chances are we would want another width

**Comment by elizabeth-jimenez** [comment]
or can make `datePickerClassName` an array of strings

**Comment by M00staff** [comment]
> Should the width maybe be a prop, cause if we use this on another page as a filter, chances are we would want another width

it _could_, but I think it would be much simpler and more flexible to pass in a className as a prop - and then customize that class as needed for each unique use case

image
image

**Comment by jlam-mpc** [comment]
> > Should the width maybe be a prop, cause if we use this on another page as a filter, chances are we would want another width
> 
> it _could_, but I think it would be much simpler and more flexible to pass in a className as a prop - and then customize that class as needed for each unique use case
> 
> image image image

You could maybe also pass in an inline styles prop so that you have unlimited reign of what width to make it if we don't want to restrict it to a fixed set of classnames

Instead of having like .width-240 .width-250 etc. you have the component setup like this:

<Component width={250} />

and in the actual component, to use the width prop:

<DatePicker
  styles={{
    width: props.width
  }}
/>

I don't like using inline styles but if something needs to be dynamic I would do that

**Review by elizabeth-jimenez — APPROVED** [review]

File: src/sass/_common.scss [hunk 1]
```diff
@@ -77,7 +77,8 @@ $values in $filters-sizing-config {
         width: $label-width;
       }
 
-      .picky {
+      .picky,
+      .tm-datepicker-wrapper {
```
Comment (M00staff) [inline src/sass/_common.scss:81]: does this need to be here?

Comment (elizabeth-jimenez) [reply src/sass/_common.scss:81]: yes

Comment (elizabeth-jimenez) [reply src/sass/_common.scss:81]: this is what sets the width - if you remove it it goes to 100%, which we may want for form inputs of date pickers, but the filters width should be set to the same as all the other filters in the set, which is what this does

Comment (M00staff) [reply src/sass/_common.scss:81]: ah yea - ok, just making sure. 

i was asking cause i saw that `.tm-datepicker-wrapper` was used both here and in the `_datepicker.scss` file

**Pull request merged**